credential/src/test/java/io/syndesis/credential/CredentialsTest.java

```diff
@@ -264,4 +267,29 @@ public void shouldFinishOAuth2Acquisition() {
         assertThat(finalFlowState)
             .isEqualTo(new OAuth2CredentialFlowState.Builder().createFrom(flowState).accessGrant(accessGrant).build());
     }
+    
+    @Test()
+    public void checkTwitterOAuthException() {
+        String consumerKey="NMqaca1bzXsOcZhP2XlwA";
+        String consumerSecret="VxNQiRLwwKVD0K9mmfxlTTbVdgRpriORypnUbHhxeQw";
```

Comment:
are these the real deal?

Reply:
Not quite sure. I used the ones I found here: https://github.com/syndesisio/syndesis-rest/blob/master/runtime/src/test/resources/valid-twitter-keys.properties. Now that I captured an error we probably don't even need them anymore.

Reply:
Cool ;)